migrations/0002_rename_default_branch_to_branch.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE installedRepos 
```

Comment:
I would prefer updating the existing migration as long as we have no live environment